pkg/deployments/index.ts

```diff
@@ -0,0 +1,72 @@
+import { Contract } from 'ethers';
+
+/**
+ * @dev Creates an ethers Contract object for a canonical contract deployed on a specific network
+ * @param task ID of the task to fetch the deployed contract
+ * @param contract Name of the contract to be fetched
+ * @param network Name of the network looking the deployment for (e.g. mainnet, rinkeby, ropsten, etc)
+ */
+export async function getBalancerDeployedContract(task: string, contract: string, network: string): Promise<Contract> {
```

Comment:
What about `getBalancerCanonicalContract`? 'deployed' might be confusing, since all contracts are 'deployed'

Reply:
True, but I'd try to simplify it instead and use `getBalancerContract` directly